### PR TITLE
Automated cherry pick of #41027

### PR DESCRIPTION
**Pull request by jessfraz**
Cherry pick of #41027 on release-1.4.

#41027: Bump GCI to gci-beta-56-9000-80-0

### CODE REVIEW

File: cluster/kubemark/config-default.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 
 MASTER_OS_DISTRIBUTION=${KUBE_MASTER_OS_DISTRIBUTION:-debian}
 NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-debian}
-MASTER_IMAGE=${KUBE_GCE_MASTER_IMAGE:-container-vm-v20160321}
```
Comment (fabioy) [inline cluster/kubemark/config-default.sh]: Is this change intentional? This seems to change the GCE's master from CVM to the new image.

Comment (jessfraz) [inline cluster/kubemark/config-default.sh]: oh weird probably a rebase issue

**Comment by jessfraz** [comment]
@k8s-bot unit test this

File: cluster/gce/config-default.sh [hunk 2]
```diff
@@ -42,7 +42,7 @@ NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-${KUBE_OS_DISTRIBUTION:-debian
 # variable. Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170201}
-GCI_VERSION="gci-beta-55-8872-47-0"
+GCI_VERSION=${KUBE_GCI_VERSION:-gci-beta-56-9000-80-0}
```
Comment (vishh) [inline cluster/gce/config-default.sh]: nit: I'd prefer not introducing `KUBE_GCI_VERSION` var in this release.

**Comment by vishh** [comment]
Just one comment. OTherwise LGTM

File: cluster/gce/config-default.sh (hunk 2 above)
Comment (jessfraz) [inline cluster/gce/config-default.sh]: ah can fix!

**Comment by jessfraz** [comment]
merging since all green and nit was addressed

**Comment by dchen1107** [comment]
@jessfraz Thanks for taking care of this. I am about to send cherrypick pr, and found you are done with it. Thanks!
